mapswipe_workers/mapswipe_workers/auth.py

```diff
@@ -1,22 +1,30 @@
 import firebase_admin
 import psycopg2
 from firebase_admin import credentials, db
-
-from mapswipe_workers.definitions import CONFIG, SERVICE_ACCOUNT_KEY_PATH
+from mapswipe_workers.config import (
+    FIREBASE_DB,
+    POSTGRES_HOST,
+    POSTGRES_PORT,
+    POSTGRES_DB,
+    POSTGRES_PASSWORD,
+    POSTGRES_USER,
+    SERVICE_ACCOUNT_KEY_PATH,
+)
+from mapswipe_workers.definitions import IMAGE_URL
 
 
 def get_api_key(tileserver: str) -> str:
     if tileserver == "custom":
         return None
     else:
-        return CONFIG["imagery"][tileserver]["api_key"]
+        return IMAGE_URL[tileserver]
```

Comment:
this should return the api key instead, but it return the url.
we need to load the api key for each tile server from `mapswipe_workers.config` above.

Reply:
Good catch. I created an IMAGE_API_KEYS dict with same structure to IMAGE_URLS to be imported from config.